lib/util/url.js

```diff
@@ -29,7 +29,7 @@ exports.resolve = require("url").resolve;
  * @returns {string}
  */
 exports.cwd = function cwd () {
-  return process.browser ? location.href : process.cwd() + "/";
+  return process.browser ? location.href : process.cwd() === "/" ? "/" : (process.cwd() + "/");
```

Comment:
This doesn't handle Windows paths.  Let's split this up into two separate steps: 1) get the current CWD, and 2) append a trailing slash if necessary.

```suggestion
  let path = process.browser ? location.href : process.cwd();

  let lastChar = path.slice(-1);
  if (lastChar === "/" || lastChar === "\\") {
    return path;
  }
  else {
    return path + "/";
  }
```

Reply:
I applied it, but browser is broken. I'm going to eagerly return when `process.browser` is truthy